prov/rxd/src/rxd_av.c

```diff
@@ -302,14 +292,12 @@ static int rxd_av_lookup(struct fid_av *av, fi_addr_t fi_addr, void *addr,
 			 size_t *addrlen)
 {
 	struct rxd_av *rxd_av;
-	fi_addr_t dg_fiaddr;
+	void *dg_fiaddr;
```

Comment:
Why switch this to void * ? Don't we insert the dg fi_addr into the util_av?

Reply:
I made a mistake there.